zebra-state/src/service/finalized_state.rs

```diff
@@ -53,6 +53,8 @@ pub struct FinalizedState {
 impl FinalizedState {
     pub fn new(config: &Config, network: Network) -> Self {
         let (path, db_options) = config.db_config(network);
+        // Note: The [`crate::constants::DATABASE_FORMAT_VERSION`] constant must
+        // be incremented each time the [`column_families`] vector changes.
```

Comment:
Not just when `column_families` changes but any part of the disk format (eg how types are serialized/deserialized) changes